charts/kube-prometheus-stack/Chart.yaml

```
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 32.2.1
+version: 32.2.2
```

new feature should raise minor version
```suggestion
version: 32.3.0
```

---

Sure, have done that.